example.yaml

```diff
@@ -167,6 +167,14 @@ read_buffer_size_bytes: 2097152
 # Sets the log level to DEBUG
 debug: true
 
+# runtime.SetMutexProfileFraction
+# Default (0) disables mutex profilinmg
```

Comment:
`profilinmg`

Reply:
you can't tell me how to spell you're not my real assigned code reviewer